test/parallel/test-stream2-httpclient-response-end.js

```diff
@@ -11,7 +11,9 @@ const server = http.createServer(function(req, res) {
     let data = '';
     res.on('readable', common.mustCall(function() {
       console.log('readable event');
-      data += res.read();
+      let input = res.read();
+      if (input)
```

Comment:
@refack Docs say:

> The 'readable' event will also be emitted once the end of the stream data has been reached but before the 'end' event is emitted.
>
> Effectively, the 'readable' event indicates that the stream has new information: either new data is available or the end of the stream has been reached. In the former case, stream.read() will return the available data. In the latter case, stream.read() will return null.

I don't know about in the context of this test, but certainly in general, `null` is a possibility.